src/main/resources/static/js/knox.js

```diff
@@ -557,6 +557,10 @@ $('#apply-operators-tooltip').click(() => {
         case endpoint.operators.MERGE:
           endpoint.designSpaceMerge(inputSpaces, outputSpace, tolerance);
           break;
+
+        case endpoint.operators.REVERSE:
+          endpoint.designSpaceReverse(inputSpaces.get(0), outputSpace);
```

Comment:
this is not valid javascript syntax

Reply:
> Oh I see it now, I was on the wrong branch. I just tested this and it threw the error
> 
> ![image](https://user-images.githubusercontent.com/7750862/62378643-42215e80-b513-11e9-967b-37843082ff79.png)
> 
> Please test all your changes before making a commit.

Sorry! I was only looking at the output in the terminal. I will remember to check the webpage console as well.